groovy-language-server/src/main/java/com/palantir/groovylanguageserver/GroovycWrapper.java

```diff
@@ -213,13 +315,53 @@ private static SymbolKind getKind(ClassNode node) {
         return SymbolKind.Class;
     }
 
-    private Set<SymbolInformation> getMethodSymbolInformations(String sourcePath, ClassNode parent, MethodNode method) {
+    private SymbolInformation getVariableSymbolInformation(String parentName, String sourcePath, Variable variable) {
+
+        Location location;
+        SymbolKind kind;
+
+        if (variable instanceof DynamicVariable) {
+            kind = SymbolKind.Field;
+            location = new LocationBuilder().uri(sourcePath).range(Ranges.createRange(-1, -1, -1, -1)).build();
+
+        } else if (variable instanceof FieldNode) {
+            kind = SymbolKind.Field;
+            location = createLocation(sourcePath, (FieldNode) variable);
```

Comment:
Yes but FieldNode extends ASTNode and implements Variable so the casting is needed on variable which is defined as its interface.